src/runners/http_service_runner.ts

```diff
@@ -71,6 +72,15 @@ export async function runHttpServiceAsync(
         logger.error(err);
     });
 
+    if (dependencies.hasSentry) {
+        const options: SentryOptions = {
+            app,
+            paths: [SRA_PATH, ORDERBOOK_PATH, META_TRANSACTION_PATH, SWAP_PATH],
```

Comment:
Is it ok that SWAP_PATH is duplicated by this runner init + the swap service runner init?